src/v3/src/components/InputPassword/InputPassword.tsx

```diff
@@ -35,8 +39,20 @@ const InputPassword: UISchemaElementComponent<{
   } = uischema.options;
   const error = messages?.value?.[0] && getMessage(messages.value[0]);
```

Comment:
So I struggled with this because initially, this error property was only for server side field level errors, but now, it supports both server side and client-side errors because client side errors are pushed into this field. 

What would you propose to rename it as if it supports both?

Reply:
name it as error should be enough, how it renders depends on what is preeent in uischema, per component  layer logic it should only depend on the uischema and be stateless if possible.